src/board_controller/brainflow_boards.cpp

```diff
@@ -79,8 +79,15 @@ BrainFlowBoards::BrainFlowBoards()
         {"package_num_channel", 0},
         {"timestamp_channel", 5},
         {"marker_channel", 6},
-        {"num_rows", 7},
-        {"other_channels", {1, 2, 3, 4}}
+        {"num_rows", 20},
+        {"other_channels", {1, 2, 3, 4}},
```

Comment:
its better to add other channels to the end, they are meaningless most of the time and when you store data in a file they will be in the beginning but will not be interesting for the users.

Also, for all boards timestamp and marker channels are in the end, lets keep it consistent

Reply:
Resolved in https://github.com/brainflow-dev/brainflow/pull/556/commits/81ac4568e73e8a23eba69f37067a4fa5ee6f4649